autopush/router/gcm.py

```diff
@@ -34,10 +34,27 @@ def __init__(self, ap_settings, router_conf):
             raise IOError("GCM Bridge not initiated in main")
         self.log.debug("Starting GCM router...")
 
-    def check_token(self, token):
+    def check_token(self, token, relo):
         if token not in self.senderIDs.senderIDs():
-            return (False, self.senderIDs.choose_ID().get('senderID'))
-        return (True, token)
+            # It is possible for some systems to allow the token to be
+            # changed, or a new token to be sent. At least one system
+            # historically allowed this. Unfortunately, that system no
```

Comment:
Could you please elaborate a bit in this comment? Which "one system historically allowed this?" What's a "mobile system"? (I don't have enough context to know if it's Fennec or the GCM router).